File: Selenium/Project/ProjectActivity7.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.testng.annotations.Test;

import java.util.List;

public class ProjectActivity7 extends BaseTest{
@Test
    public void coursesTest(){
    WebElement allCourses = driver.findElement(By.xpath("//a[@href='https://alchemy.hguy.co/lms/all-courses/']"));
    allCourses.click();
    List<WebElement> coursesList = driver.findElements(By.xpath("//div/h3[@class='entry-title']"));
    System.out.println("Number of Courses on the site are : "+coursesList.size());
}
}

File: PythonActivities/Project_Activity7.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By

def Project_Activity6():
    driver = webdriver.Chrome()
    driver.get("https://alchemy.hguy.co/lms/")
    # myAccountLink = driver.find_element(By.XPATH,"//a[@href='https://alchemy.hguy.co/lms/my-account/']").click()
    time.sleep(10)
    allCourserLink = driver.find_element(By.XPATH,"//a[@href='https://alchemy.hguy.co/lms/all-courses/']").click()

    courserList = driver.find_elements(By.XPATH, "//div/h3[@class='entry-title']")
    courserListSize = len(courserList)
    print(courserListSize)

Project_Activity6()